src/main/java/com/fasterxml/jackson/module/jsonSchema/JsonSchema.java

```diff
@@ -299,8 +335,8 @@ public String getId() {
 		return extendsextends;
 	}
 
-	public Boolean getRequired() {
-		return required;
+	public Boolean getRequiredBoolean() {
```

Comment:
Why would the type be included in accessor name? Is that for schema v4?

Reply:
Since I added the v4 version of required into the Objects hema I couldn't have 2 getRequired methods retuning different types. So I changed this one and that is the best I came up with at the time. I probably should put this back to getRequired and rename the one in ObjectSchema to requiredFields and annotate it to be @JsonProperty("required"). Thoughts?